package.json

```diff
@@ -99,9 +121,14 @@
     "redux-persist": "^6.0.0",
     "rn-nodeify": "^10.3.0",
     "stream-browserify": "^3.0.0",
+    "string_decoder": "^0.10.31",
     "text-encoding-polyfill": "^0.6.7",
+    "timers-browserify": "^1.4.2",
+    "tty-browserify": "^0.0.0",
     "url": "^0.10.3",
+    "use-memo-one": "^1.1.2",
     "util": "^0.10.4",
+    "vm-browserify": "^0.0.4",
```

Comment:
can you check all the new libs added to see if they are all needed to the final solution?